packages/test/src/test-dependencies.ts

```diff
@@ -165,16 +164,12 @@ if (RUN_INTEGRATION_TESTS) {
   });
 
   test('Worker wraps ignored dependencies and logs when they throw', async (t) => {
```

Comment:
Because that instance will be shared with other tests, no? And as a result the collected logs will have all sorts of other stuff?

Reply:
It doesn't matter because we filter specific messages that are only emitted from this test.
And if we want to avoid conflicts we can use `test.sync`.